action.yml

```diff
@@ -52,14 +59,23 @@ runs:
 
     - name: Install mkdocs and mkdocs plugins
       shell: bash
-      run: python -m pip install mkdocs-techdocs-core==0.*
+      run: python -m pip install mkdocs-techdocs-core==1.*
+
+    - name: Install additional plugins
+      if: inputs.additional-plugins != ''
+      shell: bash
+      run: |
+        for p in ${{ inputs.additional-plugins }}
+        do
+          python -m pip install $p
+        done
 
     - name: Generate docs site
       shell: bash
       run: techdocs-cli generate --no-docker --verbose
 
     - name: Publish TechDocs site to AWS S3
-      if: inputs.publisher-type == 'awsS3' && inputs.aws-access-key-id != '' && inputs.aws-secret-access-key != '' && inputs.aws-region != ''
+      if: inputs.skip-publish == 'false' && inputs.publisher-type == 'awsS3' && inputs.aws-access-key-id != '' && inputs.aws-secret-access-key != '' && inputs.aws-region != ''
```

Comment:
I would check with `true` because for now we could also write `blabla` and it would work. 😄

Reply:
Had the same thought. But if you revert the statement to `skip-publish!='true'`, then this step is executed for value `blabla`. Both not perfect :)